web/pgadmin/dashboard/static/js/SystemStats/Summary.jsx

```diff
@@ -0,0 +1,324 @@
+import React, { useState, useEffect, useRef, useReducer, useMemo } from 'react';
+import gettext from 'sources/gettext';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import url_for from 'sources/url_for';
+import getApiInstance from 'sources/api_instance';
+import {getGCD, getEpoch} from 'sources/utils';
+import {ChartContainer} from '../Dashboard';
+import { Grid } from '@material-ui/core';
+import { DATA_POINT_SIZE } from 'sources/chartjs';
+import StreamingChart from '../../../../static/js/components/PgChart/StreamingChart';
+import {useInterval, usePrevious} from 'sources/custom_hooks';
+import axios from 'axios';
+import { getStatsUrl, transformData,statsReducer, X_AXIS_LENGTH } from './Utility';
+
+const useStyles = makeStyles((theme) => ({
+  autoResizer: {
+    height: '100% !important',
+    width: '100% !important',
+    background: theme.palette.grey[400],
+    padding: '7.5px',
+    overflowX: 'auto !important',
+    overflowY: 'hidden !important',
+    minHeight: '100%',
+    minWidth: '100%',
+  },
+  table: {
+    width: '100%',
+    backgroundColor: theme.otherVars.tableBg,
+    border: '1px solid rgb(221, 224, 230)',
```

Comment:
do not use hardcoded colors, use from theme. Why do want to customise table css? It should not be required I think. We have defined table styles which are used everywhere - `const tableClasses = commonTableStyles();`